<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vecWidget</class>
 <widget class="QWidget" name="vecWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>110</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>15</y>
     <width>221</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>93</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>修改</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>93</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>定义</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
